checkov/terraform/checks/graph_checks/aws/RDSEnableCopyTagsToSnapshot.yaml

```diff
@@ -0,0 +1,40 @@
+metadata:
+  id: "CKV2_AWS_60"
+  name: "Ensure RDS instance with copy tags to snapshots is enabled"
+  category: "GENERAL_SECURITY"
+definition:
+  and:
+    - cond_type: attribute
+      resource_types: "aws_db_instance"
+      attribute: "copy_tags_to_snapshot"
+      operator: "jsonpath_equals"
+      value: "true"
+
+    - cond_type: attribute
+      resource_types: "aws_db_instance"
+      attribute: "copy_tags_to_snapshot"
+      operator: "jsonpath_exists"
+
+    - cond_type: attribute
+      resource_types: "aws_db_instance"
+      attribute: "copy_tags_to_snapshot"
+      operator: "jsonpath_exists"
```

Comment:
I think this is a duplicate and it actually it is not needed

Reply:
Oh okay. @gruebel Can you please share the Checkov ID(s)? we'll map them in PC.

Reply:
> Oh okay. @gruebel Can you please share the Checkov ID(s)? we'll map them in PC.

Got it. Working on it